### DISCUSSION TITLE
How to customize a voice?

### DISCUSSION THREAD

**Discussion by AIhasArrived** [coqui-ai/TTS#3165] [General Q&A]
So let's say I don't wanna clone a voice 100% the same, 
1) Can I mix 2 voices and expect to get good results or will it disturb the "trainin process" and degrade the results?
2) Can I add some "musics" or "sounds" in the training data to "ALTER" the voice I want to create in some ways? Like making it more melodic, more deep, etc etc
Essenstially finding ways to CONTROL and ALTER how a voice will be outputed so you will avoid produced 100% of the voice a person, instead have one custom and unique?

Any thoughts and tips/tricks to know? Thank you!

**Comment by erogol** [comment]
Adding music and sound is a bad idea. You can mix two voices with XTTSv2 by simply giving samples per speaker.

**Reply by AIhasArrived** [reply]
THis ? https://huggingface.co/coqui/XTTS-v2 ?

**Reply by erogol** [reply]
yes

**Reply by AIhasArrived** [reply]
What does mean "givine samples per speaker"? I actually not using coqui or xtts v2 at the mean time, i am using something else but I am curious
